apps/zbugs/docker/1gb_upstream/init.sql

```diff
@@ -0,0 +1,379 @@
+DROP TABLE IF EXISTS "user",
```

Comment:
is it possible to share the 1gb upstream with zbugs upstream? I would have expected zbugs and rocinante to have the same schema but maybe I'm mistaken?

Reply:
I DRYd it so we can share the same base.sql